templates/web/base/questionnaire/index.html

```diff
@@ -1,18 +1,37 @@
 [%
-    pre_map = INCLUDE 'report/_main.html';
+    SET bodyclass = 'mappage';
+    PROCESS "report/photo-js.html";
```

Comment:
Because you're including this from a template in the `base` cobrand, you'll need to move `photo-js.html` from the `fixmystreet` cobrand to `base` too.